rt/jvm/src/V3S_System.java

```diff
@@ -429,4 +429,151 @@ public static boolean feq(float a, float b) {
     public static boolean deq(double a, double b) {
 	return Double.doubleToRawLongBits(a) == Double.doubleToRawLongBits(b);
     }
+
+    public static byte readBytes1(byte[] bytes, int offset, int staticOffset) {
+        return bytes[offset + staticOffset];
+    }
+
+    public static short readBytes2(byte[] bytes, int offset, int staticOffset) {
+        int byte0 = ((int)bytes[offset + staticOffset] & 0xFF);
```

Comment:
What about factoring out `offset + staticOffset` into a (short) temp? That shortens the source a bit and will also save some bytecode space.